dev/src/main/resources/META-INF/plugin.xml

```diff
@@ -14,8 +14,50 @@
     <vendor email="codewind-dev@eclipse.org" url="https://www.eclipse.org/codewind/">Codewind</vendor>
 
     <description><![CDATA[
-        Adds support for developing cloud-native, containerized web applications.
-    ]]>    </description>
+      <h2>Codewind</h2>
+      <p>
+        Adds tools to help build cloud-native, containerized Java
+        applications. Create an application from a template project
+        and launch, update, test, and debug in containers on the
+        desktop. Supports Codewind version 0.10.0. For more
+        information, see <a href="https://www.eclipse.org/codewind/">https://www.eclipse.org/codewind/</a>.
```

Comment:
Oh, we can add hyperlinks? In that case, maybe say, `...see the <a href="https://www.eclipse.org/codewind/">Codewind website</a>` instead?

Reply:
Changed